cmds/core/gzip/gzip.go

```diff
@@ -79,4 +68,30 @@ func main() {
 			continue
 		}
 	}
+
+	return nil
+}
+
+func main() {
+	var opts gzip.Options
+	cmdLine.Usage = usage
+
+	if err := opts.ParseArgs(os.Args, cmdLine); err != nil {
+		if errors.Is(err, gzip.ErrStdoutNoForce) {
+			fmt.Fprintf(os.Stderr, "%s\n", err)
+			os.Exit(1)
+		}
+		if errors.Is(err, gzip.ErrHelp) {
+			cmdLine.Usage()
+			os.Exit(0)
+		}
+		fmt.Fprintf(os.Stderr, "%s\n", err)
+		cmdLine.Usage()
```

Comment:
I wonder if cmdline.Usage might take an io.Writer parameter, and an err parameter? Could all these lines be collapsed into a slightly more capable Usage?